tests/integration-tests/test-suites/initiateDataRequestDataCopy.spec.ts

```diff
@@ -173,27 +176,27 @@ describe('Data should be copied to analysis bucket', () => {
     let ticketId: string
 
     beforeEach(async () => {
-      await deleteAuditDataWithPrefix(
-        AUDIT_BUCKET_NAME,
-        `firehose/${INTEGRATION_TEST_DATE_PREFIX_MIX_DATA}`
-      )
-      await deleteAuditDataWithPrefix(
-        ANALYSIS_BUCKET_NAME,
-        `firehose/${INTEGRATION_TEST_DATE_PREFIX_MIX_DATA}`
-      )
-      await copyAuditDataFromTestDataBucket(
-        AUDIT_BUCKET_NAME,
-        `firehose/${INTEGRATION_TEST_DATE_PREFIX_MIX_DATA}/01/${TEST_FILE_NAME}`,
+      const availableDate = await getAvailableTestDate()
+      console.log(availableDate)
+
+      const response = await copyAuditDataFromTestDataBucket(
+        getEnv('AUDIT_BUCKET_NAME'),
+        `${availableDate.prefix}/01/${TEST_FILE_NAME}`,
         TEST_FILE_NAME,
-        'GLACIER'
+        'GLACIER',
+        true
       )
+
+      console.log(response)
```

Comment:
Is it worth logging here?  We're not catching errors from copyAuditDataFromTestDataBucket so if we get here it's worked.

Reply:
Fixed